conf/test_full.config

```diff
@@ -0,0 +1,22 @@
+/*
+ * -------------------------------------------------
+ *  Nextflow config file for running full-size tests
+ * -------------------------------------------------
+ * Defines bundled input files and everything required
+ * to run a full size pipeline test. Use as follows:
+ *   nextflow run nf-core/dualrnaseq -profile test_full,<docker/singularity>
+ */
+
+params {
+  config_profile_name = 'Full test profile'
+  config_profile_description = 'Full test dataset to check pipeline function'
+
+  // Input data for full size test
+  // TODO nf-core: Specify the paths to your full test data ( on nf-core/test-datasets or directly in repositories, e.g. SRA)
+  // TODO nf-core: Give any required params for the test so that command line flags are not needed
+  single_end = false
+  input_paths = [
+    ['Testdata', ['https://github.com/nf-core/test-datasets/raw/exoseq/testdata/Testdata_R1.tiny.fastq.gz', 'https://github.com/nf-core/test-datasets/raw/exoseq/testdata/Testdata_R2.tiny.fastq.gz']],
+    ['SRR389222', ['https://github.com/nf-core/test-datasets/raw/methylseq/testdata/SRR389222_sub1.fastq.gz', 'https://github.com/nf-core/test-datasets/raw/methylseq/testdata/SRR389222_sub2.fastq.gz']]
```

Comment:
This test is working?

Reply:
Good spotting, I don't think so. Just for clarification, this is to test the pipeline on larger datasets right?
Then, what's the best protocol to acquire the underlying raw data (as hosting on GitHub is obviously not an option).
I see that I can use SRA links somehow? (https://github.com/nf-core/rnaseq/blob/master/conf/test_sra.config). 
Would this be the prefered approach?

Reply:
Yes, that should work.